src/test/kotlin/no/nb/mlt/wls/synq/model/SynqModelValidationTest.kt

```diff
@@ -35,14 +35,33 @@ class SynqModelValidationTest {
 
     @Test
     fun `valid SynqOrderPickingConfirmationPayload should pass validation`() {
-        thenCode(validSynqOrderPickingConfirmationPayload::validate).doesNotThrowAnyException()
+        thenCode {
+            validSynqOrderPickingConfirmationPayload.validate()
```

Comment:
Maybe go back to the old test calls since validate doesn't take any input anymore